pyscriptjs/src/python/pyscript/_pyodide_compat.py

```diff
@@ -0,0 +1,7 @@
+try:
+    from pyodide.code import eval_code
+    from pyodide.ffi import JsProxy, create_once_callable, create_proxy
+except ImportError:
+    from pyodide import JsProxy, create_once_callable, create_proxy, eval_code
+
+__all__ = ["JsProxy", "create_proxy", "eval_code", "create_once_callable"]
```

Comment:
I think that this code is needed for compatibility with older versions of pyodide but we no longer support them, so maybe we can kill this?